jar-service/src/main/java/com/sixsq/slipstream/application/RootApplication.java

```diff
@@ -387,7 +387,7 @@ private void attachMetering(RootRouter router) throws ConfigurationException, Va
 
 	private void attachSSCLJ(RootRouter router) throws ValidationException {
 		for (String sscljResourceName : SSCLJRouter.SSCLJ_RESOURCE_NAMES) {			
-			guardAndAttach(router, new SSCLJRouter(getContext(), sscljResourceName), sscljResourceName);			
+			guardAndAttach(router, new SSCLJRouter(getContext(), sscljResourceName), "api/" + sscljResourceName);
```

Comment:
1) yes, this can be improved. 
2) this is a good catch, but for the other pull-request.